client/src/elements/Prompt.jsx

```diff
@@ -21,33 +32,57 @@ export function Prompt({ file, name, shared }) {
   const progressLabel = player.get("progressLabel");
   const { text: promptString, error: fetchError } = useText({ file });
   const permalink = usePermalink(file);
+
   const [responses, setResponses] = React.useState([]);
 
   if (fetchError) {
     return <p>Error loading prompt, retrying...</p>;
   }
-  if (!promptString) return <p>Loading prompt...</p>;
+
+  if (!promptString) return <p>Prompt file is empty</p>;
```

Comment:
Sorry I didn't get to review this last week. Sometimes there will not be a prompt string in the first render because the useText hook hasn't returned yet (it's async). Probably the thing to do is to check if promptString is actually an empty string (and check that useText will return an empty string if that's what it resolves), and only then say that the prompt file is empty. If it's `undefined`, return "Loading prompt..."